### PR TITLE
Completely removing leaflet-hash module

### LINKED ISSUE #509: Completely remove leaflet-hash .
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### Problem

We have removed leaflet-hash from `bower.json` but still we are using leaflet-hash functions . Lets completely remove leaflet-hash as we would be soon incorporating another library (leaflet-fullhash) to maintain uniformity across our different projects .

### 📋Solution

Let's Remove these lines :

https://github.com/publiclab/mapknitter/blob/05cb5593f4c1b6cb0e786046da1cf82117359489/app/views/maps/index.html.erb#L5

**and**

https://github.com/publiclab/mapknitter/blob/05cb5593f4c1b6cb0e786046da1cf82117359489/app/views/maps/index.html.erb#L108

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update**

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by alaxalves**
Fixes #509  (<=== Add issue number here)
Just removed the indicated lines in #509 in order to remove leaflet-hash dependencie.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by alaxalves** [comment]
@publiclab/reviewers could you check this out?

**Comment by gauravano** [comment]
Hi @sagarpreet-chadha, this can be merged right?  Also, this seems relevant https://github.com/publiclab/mapknitter/pull/516. Thanks!

**Comment by jywarren** [comment]
OK, oops, i just merged #516 and looks like that's supposed to replace this one? I caused a conflict, sorry!

**Comment by alaxalves** [comment]
> OK, oops, i just merged #516 and looks like that's supposed to replace this one? I caused a conflict, sorry!

Fixed it

File: app/views/maps/index.html.erb [hunk 1]
```diff
@@ -2,7 +2,6 @@
 <%= stylesheet_link_tag "/lib/leaflet-environmental-layers/dist/LeafletEnvironmentalLayers.css" %>
 <%= javascript_include_tag('/lib/leaflet-environmental-layers/dist/LeafletEnvironmentalLayers.js') %>
 <%= javascript_include_tag('/lib/leaflet-environmental-layers/dist/windRoseLayer.js') %>
-<%= javascript_include_tag('/lib/leaflet-fullhash/leaflet-fullHash.js') %>
```
Comment (sashadev-sky) [inline app/views/maps/index.html.erb]: This is `leaflet-fullhash` not `leaflet-hash`

Comment (alaxalves) [reply app/views/maps/index.html.erb]: Ohh, sorry for this @sashadev-sky, I'll fix it. :astonished:

**Comment by sashadev-sky** [comment]
@alaxalves try replacing the line exactly where you originally deleted it from so that this file is removed all together from the "files changed" view

**Comment by alaxalves** [comment]
> @alaxalves try replacing the line exactly where you originally deleted it from so that this file is removed all together from the "files changed" view

Done! :smile:

**Comment by sashadev-sky** [comment]
@jywarren this one is ready for merge!

**Comment by jywarren** [comment]
Thanks!!!

**Pull request merged**